resources/Makefile.j2

```diff
@@ -0,0 +1,111 @@
+	test -d configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/ || cp -rf {{ directories.phenotype }} configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/
+	java -cp $(H2_JAR) org.h2.tools.RunScript -user sa -url jdbc:h2:$(shell pwd)/configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/{{ phenotype.version }}_phenotype  -script $<
+
+semsim-ingest: configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/{{ phenotype.version }}_phenotype.h2.db
+
+{% endif %}
+
+{% endfor %}
+
+.PHONY: prepare-corpora
+
+{% for run in runs %}
+results/{{ run.tool }}-{{ run.version }}-{{ run.configuration }}/{{ run.corpus }}-scrambled{{ run.corpusvariant }}/results.yml: configurations/{{ run.tool }}-{{ run.version }}-{{ run.configuration }}/config.yaml
+	
+	pheval run \
+	 --input-dir {{ directories.config }}/{{ run.tool }}-{{ run.version }}-{{ run.configuration }} \
+	 --testdata-dir $(shell pwd)/corpora/{{ run.tool }}/{{ run.corpus }}/scrambled{{ run.corpusvariant }} \
+	 --runner {{ run.tool }}phevalrunner \
+	 --tmp-dir {{ directories.tmp }}/ \
+	 --output-dir $(shell dirname pwd)/$(shell dirname $@)
+	
+	touch $@
+
+
+corpora/{{ run.tool }}/{{ run.corpus }}/scrambled{{ run.corpusvariant }}/corpus.yml: $(TEST_DATA)/template_vcf/template_exome_hg19.vcf.gz
+	{% if run.tool == 'exomiser' %}
+	test -d $(shell dirname $@)/vcf || mkdir -p $(shell dirname $@)/vcf
+	test -L $(shell pwd)/corpora/{{ run.tool }}/{{ run.corpus }}/scrambled{{ run.corpusvariant }}/template_exome_hg19.vcf.gz || ln -s $(shell pwd)/$< $(shell dirname $@)/vcf/
+	pheval-utils create-spiked-vcfs \
+	 --template-vcf-path $(shell pwd)/$(shell dirname $@)/vcf/template_exome_hg19.vcf.gz \
+	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/single \
+	 --output-dir $(shell pwd)/$(shell dirname $@)/vcf
+	{% endif %}
+
+	test -d $(shell dirname $@)/phenopackets || mkdir -p $(shell dirname $@)/phenopackets
+	pheval-utils scramble-phenopackets \
+	 --scramble-factor {{ run.corpusvariant }} \
+	 --output-dir $(shell pwd)/$(shell dirname $@)/phenopackets \
+	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/single
+	touch $@
+
+
+prepare-corpora: corpora/{{ run.tool }}/{{ run.corpus }}/scrambled{{ run.corpusvariant }}/corpus.yml
+
+
+
+run-{{ run.corpus }}-{{ run.tool }}:
+	$(MAKE) results/{{ run.tool }}-{{ run.version }}-{{ run.configuration }}/{{ run.corpus }}-scrambled{{ run.corpusvariant }}/results.yml
+
+pheval-run: run-{{ run.corpus }}-{{ run.tool }}
+
+{% endfor %}
+
+
+.PHONY: pheval
+pheval: 
+	$(MAKE) prepare-inputs
+	$(MAKE) prepare-corpora
+	$(MAKE) pheval-run
+
+
+.PHONY: clean
+clean:
+	rm -rf data/* corpora/* inputs/* configurations/* results/*
```

Comment:
what is in "data"?

Don't clean "corpora" - these are part of the framework!

Reply:
So, I'll remove this goal, because there's no folder left to remove (they're included in .gitignore file)